parsers/nmparser/utils.go

```diff
@@ -20,25 +23,29 @@ func createDiagonalBlock(num int) []int {
 // Checks the final estimation method against a list of non-gradient based methods.
 // Primarily used for deciding whether to look for a .grd file to parse
 func CheckIfNotGradientBased(results SummaryOutput) bool {
-	isNotGradientBased := funk.Contains([]string{
-		"MCMC Bayesian Analysis",
+		finalMethod := results.RunDetails.EstimationMethods[len(results.RunDetails.EstimationMethods)-1]
```

Comment:
This is unrelated too, but @timwaterhouse 's model had
```
Estimation Method(s):
 - Stochastic Approximation Expectation-Maximization (No Prior)
 - Objective Function Evaluation by Importance Sampling (No Prior)
```
neither of which were triggering `isNotGradientBased` previously. This seemed like a better solution going forward.

Reply:
what is the concept here? This is now changing it if any of the methods are non-gradient based rather than the last one only right?

Reply:
No it's that previously we were doing exact string matching which meant that "Importance Sampling" was included but "Importance Sampling (No Prior)" didn't match. Now we're doing substring matching so we don't have to check as many things and also because NONMEM seems to have a lot of different variations on what it prints out here.

Reply:
To be clear, we are still only checking the final method. That is correct, right?